test/connectors/CustomsDeclareExportsConnectorISpec.scala

```diff
@@ -245,26 +245,6 @@ class CustomsDeclareExportsConnectorISpec extends ConnectorISpec with ExportsDec
     }
   }
 
-  "Find Declarations" should {
```

Comment:
Why has this been removed?

Reply:
Because these tests were using `CustomsDeclareExportsConnector.findDeclarations` which was an unused method.